src/command/vanilla/tell-command.js

```diff
@@ -20,25 +16,25 @@ class TellCommand extends Command {
      */
     execute(sender, args) {
         if (!args[0]) {
-            return sender.sendMessage('§cYou have te select a player.')
+            return sender.sendMessage('§cYou have to select a player.')
         }
 
         if (`${args[0]}`.toLowerCase() == sender.name.toLowerCase()) {
             return sender.sendMessage('§cYou can\'t send message to yourself.')
         } 
 
         if (!args[1]) {
-            return sender.sendMessage('§cInvalid message.')
+            return sender.sendMessage('§cPlease specify a message.')
         } 
 
         let targetPlayer = sender.getServer().getPlayerByName(args[0])
         if (!targetPlayer) {
-            return sender.sendMessage(`§cCan not find the player ${args[0]}.`)
+            return sender.sendMessage(`§cCan't find the player ${args[0]}.`)
         }
 
         let message = args.splice(1).join(' ')
-        sender.sendMessage(`§7§oMe -> ${targetPlayer.name}: ${message}`)
-        targetPlayer.sendMessage(`§7§o${sender.name} -> Me: ${message}`)
+        sender.sendMessage(`§e[§f${sender.name} §e->§f ${targetPlayer.name}§e]§f ${message}`)
```

Comment:
and you can make that like it:

**let messageToSend = `§e[§f${sender.name} §e->§f ${targetPlayer.name}§e]§f ${message}`**
**sender.sendMessage(messageToSend)**
**targetPlayer.sendMessage(messageToSend)**

for performance improvement